### PR TITLE
Added declarative pipeline example

### LINKED ISSUE #322: [documentation] show example with Declarative syntax
### Describe your use-case which is not covered by existing documentation.

please show example with Declarative syntax

### Reference any relevant documentation, other materials or issues/pull requests that can be used for inspiration.

_No response_

### PR DESCRIPTION
**Pull request by Jagrutiti**
Close #322 

### Testing done

NN

### Proposed upgrade guidelines

N/A

### Localizations

- [x] English
- [ ] Franc
- [ ] Slovak
- [ ] Czech
- [ ] ...

### Submitter checklist

- [x] The Jira / Github issue, if it exists, is well-described.

### Maintainer checklist

Before the changes are marked as `ready-for-merge`:

- [x] There is at least one (1) approval for the pull request and no outstanding requests for change.
- [x] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [x] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [x] Proper changelog labels are set so that the changelog can be generated automatically.

### CODE REVIEW

**Comment by jimklimov** [comment]
As noted on Gitter, I think it makes better sense to not convert but to clearly mark "Example for scripted pipeline" vs. "Example for declarative pipeline" since both are actively used, as different tools for different jobs.

Also, try to constrain the initial PR to focus on its feature changes. Seems it currently includes a version of commits already merged (or proposed elsewhere) to master and they show up in https://github.com/jenkinsci/lockable-resources-plugin/pull/412/files as changes for review: git-rebasing over current upstream/master can help.

**Comment by Jagrutiti** [comment]
Hey @jimklimov 

I have added your suggestions and re-based the commit to master.

**Comment by jimklimov** [comment]
Thanks for the rebase, still the "Changes" look odd. Was `[PULL_REQUEST_TEMPLATE.md](https://github.com/jenkinsci/lockable-resources-plugin/pull/412/files#diff-479755504b0aad0fed26047714591511b4d07c76bc8521171b87a5f16100f850)` and `[org/jenkins/plugins/lockableresources/actions/LockableResourcesRootAction/index.properties](https://github.com/jenkinsci/lockable-resources-plugin/pull/412/files#diff-812cc60f681b8439c0770757ab5795b69af89070a3c0a5c3e6cd95716821056f)` intentional? Or just grabbed by git from that other branch as a difference vs. master?

**Comment by mPokornyETM** [comment]
When I am correct , this branch was created from `extend-documentation`
That explain the changes

**Comment by mPokornyETM** [comment]
I propose to wait for this PR #403 and then re-merge master to this branch. After them you will see real changes.
@Jagrutiti thx for your time

**Comment by Jagrutiti** [comment]
Hey @mPokornyETM , @jimklimov 

I had created my branch from `extend-documentation`. I am using this branch so that we do not face merging issues. [I had this discussion with @mPokornyETM.](https://github.com/jenkinsci/lockable-resources-plugin/issues/322#issuecomment-1321134534)

I wanted you guys to confirm:

1. Is my script example correct? 
2. Is it alright if I do that same for other examples in the `read.me` file?
3. Which branch shall I use as base branch to make my next PR?

**Comment by mPokornyETM** [comment]
Thx I will wait until #403 is merged.  Then it will be easier to review the changes.  Pls wait next 1 or 2 days and I give you here a feedback.  Thx

**Review by mPokornyETM — CHANGES_REQUESTED** [review]

File: README.md
Comment (mPokornyETM) [inline README.md:92]:
```suggestion
```

**Review by mPokornyETM — APPROVED** [review]
LGTM

**Pull request merged**
